packages/order-utils/src/transformer_data_encoders.ts

```diff
@@ -147,17 +147,11 @@ export function decodePayTakerTransformerData(encoded: string): PayTakerTransfor
  */
 export const affiliateFeeTransformerDataEncoder = AbiEncoder.create({
```

Comment:
All this needs to be reverted to original. I goofed. `AbiEncoder` can't handle this kind of encoding for some reason and it ends up reverting.